Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Init.hs

```diff
@@ -151,17 +167,72 @@ handler merchantId req eitherReq = do
             Nothing -> logDebug "Unable to get the key"
 
           return (booking, Nothing, Nothing)
-        Left _ -> throwError $ InvalidRequest "Can't have driverQuote in specialZone booking"
+        _ -> throwError $ InvalidRequest "Can't have driverQuote in specialZone booking"
+    InitRentalReq -> do
+      case initReq of
+        RENTAL_QUOTE (rentalQuote, searchRequest) -> do
+          (duration :: Int) <- case req.rentalDuration of
+            Nothing -> throwError $ InvalidRequest "Request Invalid"
```

Comment:
Will change the err message..

This is the part of Init request to BPP. So while parsing if we don't get rentalDuration then it can be InvalidRequest I guess  as a required field  is not send in request from BAP side ?